### PR TITLE
fix tslint config

### PR DESCRIPTION
**Pull request by BO41**
both `tslint.json`s show warnings about a wrong configuration

https://palantir.github.io/tslint/rules/member-ordering/

https://palantir.github.io/tslint/rules/indent/

### CODE REVIEW

**Comment by rigelk** [comment]
I'm still getting a warning `Warning: member-ordering - Bad member kind: public-before-private` with your modifications.

File: client/tslint.json [hunk 1]
```diff
@@ -7,10 +7,12 @@
...
-    "member-ordering": [true,
-      "public-before-private",
-      "static-before-instance",
-      "variables-before-functions"
+    "member-ordering": [true, {
+      "order": [
+        "public-before-private",
+        "static-before-instance",
+        "variables-before-functions"
+        ]}
```
Comment (rigelk) [inline client/tslint.json]: A similar syntax by the tslint docs should be:

```
"public-static-field",
"public-instance-field",
"public-constructor",
"private-static-field",
"private-instance-field",
"private-constructor",
"public-instance-method",
"protected-instance-method",
"private-instance-method"
```

Comment (BO41) [reply client/tslint.json]: I can't reproduce the warnings you get. my ide seems not to show them.
But you are right, according to the docs `"public-before-private"` and the others are not allowed, yet they still seem to work.

Should I change it to your proposal or to something of the examples?

Comment (rigelk) [reply client/tslint.json]: Try with a terminal and `npm run travis -- lint`.

Yes, I've not tried this proposal but it should be close enough :crossed_fingers:

**Comment by rigelk** [comment]
Thanks @BO41

P.S.: I'll let @Chocobozzz answer on that one :sweat_smile:

**Comment by Chocobozzz** [comment]
> quick question btw:

It's because we share the same template between `user-create` and `user-update`

**Pull request merged**